docs/build_parameters.rst

```diff
@@ -83,6 +83,7 @@ Example of **REACTOR_CONFIG**::
         root_url: https://koji.example.com/root
         auth:
             ssl_certs_dir: /var/run/secrets/atomic-reactor/kojisecret
+        use_fast_upload: true
```

Comment:
should we recommend `True` here? Is that only current fedora-infra workaround or a preferred way of uploading?

Reply:
I am honestly not sure, but `use_fast_upload` defaults to true in the koji.conf you get when installing the koji client. 
Generally I think setting it to true will not hurt :smile:, but I am happy to change that to `false` if needed.

Reply:
I'd actually be happy for us to default to fast-upload, i.e. for this option to have default value `true`.

Reply:
ACK